libsplinter/src/registry/mod.rs

```diff
@@ -245,9 +245,7 @@ pub trait RegistryReader: Send + Sync {
     /// # Arguments
     ///
     ///  * `identity` - The identity of the node.
-    fn has_node(&self, identity: &str) -> Result<bool, RegistryError> {
```

Comment:
I forgot that you could do this with traits; I updated the commit to leave the default implementation and just override the implementations for local YAML and unified registries (since the overriding implementations are more efficient).